base/loggers/Loggers.cpp

```diff
@@ -247,11 +247,8 @@ void Loggers::updateLevels(Poco::Util::AbstractConfiguration & config, Poco::Log
     if (log_level > max_log_level)
         max_log_level = log_level;
 
-    const auto log_path = config.getString("logger.log", "");
```

Comment:
Can you elaborate on how this commits to the following?
> Allow only to specify --log-file without configuration in logger.log

Reply:
If you don't speficy `logger.log` in `config.xml` while instead using `--log-file` in cmdline, it will always get empty string and reset the logging level to zero immediately during startup.

Reply:
Yep, I was talking about description of the PR itself, since you are still allow to use `--log-file` and it will be applied, the only difference is that it will be reset anymore in case of empty `logger.log` in config

So maybe something like this will be better?

> Do not reset logging that configured via `--log-file`/`--errorlog-file` in case of empty `logger.log`/`logger.errorlog`